.github/actions/setup-sentry/action.yml

```
@@ -7,8 +7,8 @@ inputs:
     default: 'false'
   kafka:
     description: 'Is kafka required?'
-    required: false
-    default: 'false'
+    required: true
```

I think we can keep this as `false`

```suggestion
    required: false
```